docker/compose-spec-l1-services.yml

```diff
@@ -78,9 +78,9 @@ services:
       - /bin/bash
       - -c
       - |
-        PRAGUE=${PRAGUE} bash /generate-genesis.sh
+        PRAGUE=${PRAGUE:-} bash /generate-genesis.sh
         /usr/local/bin/eth2-testnet-genesis deneb \
-        --config /config/network-config.yml \${L1_GENESIS_TIME:+--timestamp ${L1_GENESIS_TIME} \}
+        --config /config/network-config.yml \${L1_GENESIS_TIME:+--timestamp ${L1_GENESIS_TIME:-} \}
```

Comment:
To address the warning logs of missing env variables when running docker compose